__tests__/server.test.js

```diff
@@ -156,30 +156,50 @@ describe("backend API project", () => {
           expect(body.articles).toBeSortedBy("votes", { descending: true });
         });
     });
-    test("that the articles are ordered by titles and in asc order when given a sort_by of titles and a order of asc", () => {
+    test("200: that the articles are ordered by titles and in asc order when given a sort_by of titles and a order of asc", () => {
       return request(app)
         .get("/api/articles?sort_by=title&order=asc")
         .expect(200)
         .then(({ body }) => {
           expect(body.articles).toBeSortedBy("title");
         });
     });
-    test("that a GET request with a sort_by that is not valid returns 400 Bad request", () => {
+    test("200: that a GET request with a topic query returns an array of article objects with the correct topic", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.articles)).toBe(true);
+          expect(body.articles.length).toBeGreaterThan(0);
+          body.articles.forEach((article) => {
+            expect(article.topic).toBe("mitch");
+          });
+        });
+    });
+    test("400: that a GET request with a sort_by that is not valid returns 400 Bad request", () => {
       return request(app)
         .get("/api/articles?sort_by=badrequest")
         .expect(400)
         .then(({ body }) => {
           expect(body.msg).toBe("Bad request");
         });
     });
-    test("that a GET request with a order that is not valid returns 400 Bad request", () => {
+    test("400: that a GET request with a order that is not valid returns 400 Bad request", () => {
       return request(app)
         .get("/api/articles?order=badrequest")
         .expect(400)
         .then(({ body }) => {
           expect(body.msg).toBe("Bad request");
         });
     });
+    test("400: that a GET request with a topic that is not valid returns 400 Bad request", () => {
+      return request(app)
+        .get("/api/articles?topic=badrequest")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
```

Comment:
Good tests so far! We're just missing one more though- What would happen if we sent a request for a valid topic that has no articles? (E.g. "paper")